server/util/networking/networking.go

```diff
@@ -57,6 +57,12 @@ func namespace(netNamespace string, args ...string) []string {
 	return append([]string{"ip", "netns", "exec", netNamespace}, args...)
 }
 
+// Deletes all of the net namespaces. These can be left behind if the executor
+// doesn't exit gracefully.
+func DeleteNetNamespaces(ctx context.Context) error {
+	return runCommand(ctx, "ip", "--all", "netns", "delete")
```

Comment:
I'm not sure that the executor will be the only thing that creates network namespaces on the host system, especially for local development and on-prem.

wdyt about creating all net namespaces under a common prefix like `bb-executor` and then deleting only the ones matching the prefix?

Reply:
Good idea, changed and added a flag to support multiple executors on a machine like we discussed.